docs/quickstart_guide.md

```diff
@@ -289,6 +345,29 @@ for benchmarking asynchronous (i.e., Knative Eventing) case and more details abo
 ### 3. Delete Deployed Functions
 **On the master node**, execute the following instructions below using **bash**:
 
+1. Delete **all** deployed functions:
+    ```bash
+    kn service delete --all
+    ```
+
+## V. Deploying eStargz-based Functions
+This section provides an example function run using a `nodejs` base image that has been converted to the `stargz` format. To create other images supported by `stargz`, please refer to the [creating-estargz-images-using-ctr-remote](https://github.com/containerd/stargz-snapshotter/tree/cmd/v0.12.1#creating-estargz-images-using-ctr-remote) section of the official `stargz` repository.
+
+Our example image can be found in [/configs/knative_workloads/stargz-node.yaml](../configs/knative_workloads/stargz-node.yaml) and can be run with:
+```bash
+kn service apply stargz-test -f configs/knative_workloads/stargz-node.yaml --concurrency-target 1
+```
+### 1. Deploy and Invoke Functions
```

Comment:
## Invoking Functions does not work - made json files using output URL

```bash
nehalem@node-0:~/vhive$ kn service apply stargz-test -f configs/knative_workloads/rnn_serving.yaml --concurrency-target 1
Applying service 'stargz-test' in namespace 'default':

  0.044s The Configuration is still working to reflect the latest desired specification.
  2.213s Traffic is not yet migrated to the latest revision.
  2.240s Ingress has not yet been reconciled.
  2.296s Waiting for load balancer to be ready
  2.530s Ready to serve.

Service 'stargz-test' applied to latest revision 'stargz-test-00003' is available at URL:
http://stargz-test.default.192.168.1.240.sslip.io
```

```json
[
        {
                "hostname": "http://stargz-test.default.192.168.1.240.sslip.io",
                "eventing": false,
                "matchers": null
        }
]
```

but does not execute properly leaving the csv file empty.

```
nehalem@node-0:~/vhive$ pushd ./examples/invoker && go build && popd && ./examples/invoker/invoker -endpointsFile ./endpoints_stargz.json
~/vhive/examples/invoker ~/vhive
~/vhive
INFO[2023-02-04T02:48:06.957642593-07:00] Reading the endpoints from the file: ./endpoints_stargz.json
INFO[2023-02-04T02:48:11.958884828-07:00] Issued / completed requests: 5, 0
INFO[2023-02-04T02:48:11.958935677-07:00] Real / target RPS: 0.00 / 1
INFO[2023-02-04T02:48:11.958958183-07:00] Experiment finished!
INFO[2023-02-04T02:48:11.958975660-07:00] The measured latencies are saved in rps0.00_lat.csv
```

## IV.2.2 On any node -> cannot run invoker in worker node: go package not installed; can run on master

`Command 'go' not found, but can be installed with:`

skipping `source /etc/profile && go build;` may be the problem


- Also, I think it will be best to augment the section about how to deploy and invoke eStargz-based Functions

Reply:
I have updated the skipping steps part to include the source bit. I am looking into why the experiment doesn't execute properly.

Could you explain what did you meant by augmenting the sections? I kept them separate following what was done before, in the standard case.

Reply:
By augmentation, I mean some more detailed explanation about deploying, invoking and changing the `json` file may be nice.

Reply:
The deployment code you gave in the quickstart guide worked well
`kn service apply stargz-test -f configs/knative_workloads/stargz-node.yaml --concurrency-target 1`

About invoking function, I used following commands:

`curl http://stargz-test.default.192.168.1.240.sslip.io`

<img width="659" alt="Screenshot 2023-02-09 at 6 24 48 PM" src="https://user-images.githubusercontent.com/46832019/217771437-327f6cd2-22dc-4961-9e6f-b01cad443f9c.png">

And it worked well!
It would be nice to mention not to use invoker in case of stargz.

Reply:
it's not about invoker not working with stargz, it's about relatively slow coldstarts that caused invoker requests to time out. I think if you ran invoker several times back to back, the responses would be there

Reply:
@NeHalem90 Thank you so much for your suggestions! They have been really helpful! I have updated the readme with the suggestions and I would appreciate if you can have a look again over it and let me know what might be missing.

Reply:
LGTM!